src/components/styled/Wrappers.tsx

```diff
@@ -95,6 +95,16 @@ export const SearchListItem = styled(ListItem)(({ theme }) => ({
   background: 'white',
 }));
 
+export const SearchListContainer = styled('div')(({ theme }) => ({
```

Comment:
Kanskje noe sånt for å tydeliggjøre at det er det samme? eller `SearchItem`, `StyledSearchItem`, elns. Tenker også siden dette er faktisk item, og ikke "container" noen liste (som jeg leser navnet nå)
```suggestion
export const SearchListItemDiv = styled('div')(({ theme }) => ({
```